src/SDKs/SqlManagement/Sql.Tests/Sql.Tests.csproj

```diff
@@ -52,4 +52,9 @@
     <Folder Include="SessionRecords\Sql.Tests.ManagedInstanceVulnerabilityAssessmentScenarioTests\" />
     <Folder Include="SessionRecords\Sql.Tests.ServerVulnerabilityAssessmentScenarioTests\" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Update="VirtualClusterCrudScenarioTests.cs">
```

Comment:
Was this intended?

Reply:
No, thanks for noticing. I removed this.